src/docfx/restore/RestoreMap.cs

```diff
@@ -44,17 +49,23 @@ public bool TryGetUrlRestorePath(string remote, out string restorePath)
             return false;
         }
 
-        public string GetUrlRestorePath(string remote)
+        public string GetUrlRestorePath(string docsetPath, string path)
         {
-            Debug.Assert(!string.IsNullOrEmpty(remote));
+            Debug.Assert(!string.IsNullOrEmpty(path));
+
+            if (!HrefUtility.IsAbsoluteHref(path))
```

Comment:
>IsAbsoluteHref [](start = 29, length = 14)

Can we have an method `HrefUtility.IsHttp`? `IsAbsolutePath` behaves differently between windows and linux. I was thinking about deprecating this method.

Reply:
add todo for that

---
In reply to: [203647447](https://github.com/dotnet/docfx/pull/3101#discussion_r203647447) [](ancestors = 203647447)